doc/index.rst

```diff
@@ -64,7 +64,7 @@ To get started with SpikeInterface, you can take a look at the following additio
 
 - | The `2020 eLife paper <https://elifesciences.org/articles/61834>`_ introduces the concept and motivation and 
   | performs an in-depth comparison of multiple sorters (spolier alert: they strongly disagree between each other!). 
-  | **Note**: the code-base and implementation have changed a lot since the "paper" version in published in 2020. 
+  | **Note**: the code-base and implementation have changed a lot since the "paper" version published in 2020. 
```

Comment:
I would probably accept this without any other step but here I would probably like to know whether @alejoe91 and @samuelgarcia want to say any further words on this.

Reply:
Yeah it possible they wanted a link to the eLife paper there? But as is it is just a floating "in". In the future I could definitely double check on those. Thanks!